build.go

```diff
@@ -36,21 +38,33 @@ func (r *Runner) Command(command string, args ...string) error {
 	return nil
 }
 
-func (r *Runner) UpdateYum() error {
-	return r.Command("yum", "update -y")
-}
+func (r *Runner) InstallPackage(packages []string) error {
+
+	if !r.updated {
+		r.Command("yum", "update -y")
+		r.updated = true
+	}
 
-func (r *Runner) InstallPackages(packages []string) error {
 	for _, p := range packages {
-		err := r.Command("yum", "install -y "+p)
-		if err != nil {
-			return err
+		if p != "update" {
```

Comment:
So in fact we never need to pass the 'fake' package name `update` to InstallPackages(), because the Yum update is handled automatically. So we can eliminate the check for `p != "update"`.

Reply:
that makes sense, just hardcode it directly into the process.